_includes/components/image-example-incorrect.html

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+include
+- image-alt-text
```

Comment:
I think it's more standard to snake_case these include names in other partials, though it'd be worth double-checking. I'd expect `image_alt_text` for example.

Reply:
Made that change. Using snake_case for all include names.